default.nix

```diff
@@ -1,7 +1,7 @@
 # This entry points allows calling `nix-build -A` with
 # anything defined in the overlay (or the host system).
 {
-  device
+  device ? builtins.getEnv "MOBILE_NIXOS_DEVICE"
```

Comment:
This should still be an error when ```builtins.getEnv "MOBILE_NIXOS_DEVICE" == ""```

Reply:
Yeah, I thought it would return an error if the variable isn't set and ran into a confusing error because of it the other day. Will fix, thanks for the reminder!